tap_s3_csv/s3.py

```diff
@@ -230,6 +239,7 @@ def sampling_gz_file(table_spec, s3_path, file_handle, sample_rate):
 def sample_file(table_spec, s3_path, file_handle, sample_rate, extension):
     global skipped_files_count
 
+    global skipped_files_count
```

Comment:
@dbshah1212 Why are we declaring same variable twice?

Reply:
It left while resolving merge conflict, removed now.